app/views/comments/_comments.html.erb

```diff
@@ -0,0 +1,28 @@
+  <h1>Comments</h1>
+  <div class="table-responsive">
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th>User</th>
+          <th>Date</th>
+          <th>body</th>
+        </tr>
+      </thead>
+      <tbody id="comment-tbody">
+        <% comments.each do |comment| %>
+          <tr id="comment-<%= comment.id %>">
+            <td><%= comment.user.user_name %></td>
+            <td><%= comment.created_at %></td>
+            <td><%= comment.body %></td>
+            <td>
+              <% if comment.user == current_user %>
```

Comment:
`comment.user == current_user`
this will make the query and then compare two `User` objects. Pls update condition to get this functionality by simply comparing ids.